### PR TITLE
Cell Centers and Volumes

### PR DESCRIPTION
**Pull request by BernardoLFaria**
There is a Cell_data.txt where is written the output

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -15,6 +15,7 @@
 #include "mathOperations.h"
 #include "Mesh.cpp"
 #include "findFiles.h"
+//#include "calCellCentersVols.cpp"
```
Comment (brunoramoa) [inline main.cpp:18]: Please remove this line

File: main.cpp [hunk 2]
```diff
@@ -38,11 +39,20 @@ int main()
     
     Mesh polyMesh;
 
-    for (unsigned int i = 0; i< polyMesh.nFaces_ ; i++)
```
Comment (brunoramoa) [inline main.cpp:41]: move this from main.cpp to the mesh.cpp

**Review by brunoramoa — CHANGES_REQUESTED** [review]
Did you check if the calculations are correct ?

**Review by brunoramoa — COMMENTED** [review]
Did you verify the calculations?

**Comment by BernardoLFaria** [comment]
> Did you verify the calculations?
I used checkMesh in OpenFOAM v2012 to get the cellVolumes and gives me this huge difference in terms of volumes. I would be grateful if you could confirm me in OpenFOAM v2106 if the cellVolumes I obtained from v2012 are indeed correct.
image

**Comment by brunoramoa** [comment]
You can obtain the cell centers with:
postProcess -func writeCellCentres

And the cell volumes with:
postProcess -func writeCellVolumes

**Comment by BernardoLFaria** [comment]
> You can obtain the cell centers with:
> postProcess -func writeCellCentres
> 
> And the cell volumes with:
> postProcess -func writeCellVolumes

I already collected the data, as also put in a SpreadSheet. However, there is the SpreadSheet always return.
image

I think is due to the way numbers are encoding/format. If you could help , I would be grateful .. I leave the document attached:[myData.ods](https://github.com/Computational-Rheology/CRheoLab/files/6823383/myData.ods) .

**Comment by BernardoLFaria** [comment]
> > You can obtain the cell centers with:
> > postProcess -func writeCellCentres
> > And the cell volumes with:
> > postProcess -func writeCellVolumes
> 
> I already collected the data, as also put in a SpreadSheet. However, there is the SpreadSheet always return.
> image
> 
> I think is due to the way numbers are encoding/format. If you could help , I would be grateful .. I leave the document attached:[myData.ods](https://github.com/Computational-Rheology/CRheoLab/files/6823383/myData.ods) .

I already correct the error . It is the formating of the numbers

**Pull request closed without merging**